modules/ocf_desktop/files/xsession/Xsession

```diff
@@ -114,7 +114,10 @@ xset m 0 0
 sleep 8 && /usr/local/bin/fix-audio &
 
 # Auto logout inactive users
-/usr/bin/xautolock -locker /usr/local/bin/auto-lock -time 7 -noclose &
+# killer is called killtime minutes after locker is called
+# any user input including shaking the mouse then not clearing the lockscreen is considered user input
```

Comment:
yeah I'm not opposed to lowering it. I just picked a number, what do you think would be a good number? I do not have strong feelings towards 90 at all.